src/AddressesServiceProvider.php

```diff
@@ -64,4 +66,9 @@ public function configureTipoffPackage(TipoffPackage $package): void
             ->name('addresses')
             ->hasConfigFile();
     }
+
+    public function boot()
```

Comment:
Better to use `booting()` method and because this is not standard provider, always call `parent::XXX` in ANY overrides (boot or booting)